util/lru/lru_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"golang.org/x/exp/maps"
```

Comment:
elsewhere we've been importing this now as xmaps:

```
% git grep -F 'xmaps "'
cmd/tailscale/cli/exitnode.go:  xmaps "golang.org/x/exp/maps"
cmd/testwrapper/testwrapper.go: xmaps "golang.org/x/exp/maps"
ipn/ipnlocal/local.go:  xmaps "golang.org/x/exp/maps"
wgengine/filter/filter_test.go: xmaps "golang.org/x/exp/maps"
wgengine/magicsock/endpoint.go: xmaps "golang.org/x/exp/maps"
wgengine/magicsock/magicsock_test.go:   xmaps "golang.org/x/exp/maps"
```

(to distinguish it from the std maps)

Reply:
Done